Shared/Samples/Add 3D tiles layer/Add3DTilesLayerView.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct Add3DTilesLayerView: View {
+    /// A scene with dark gray basemap and an OGC 3D tiles layer.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene and sets an initial viewpoint.
+        let scene = Scene(basemapStyle: .arcGISDarkGray)
+        let camera = Camera(
+            latitude: 48.84553,
+            longitude: 9.16275,
+            altitude: 350,
+            heading: 0,
+            pitch: 75,
+            roll: 0
+        )
+        scene.initialViewpoint = Viewpoint(boundingGeometry: camera.location, camera: camera)
+        
+        // Creates a surface and adds an elevation source.
+        let surface = Surface()
+        surface.addElevationSource(ArcGISTiledElevationSource(url: .worldElevationService))
+        
+        // Sets the surface to the scene's base surface.
+        scene.baseSurface = surface
+        
+        // Adds a OGC 3D tiles layer from a URL to the scene's operational layers.
+        scene.addOperationalLayer(OGC3DTilesLayer(url: .stuttgart3DTiles))
```

Comment:
```suggestion
        // some comment...
        let ogc3DTilesLayer = OGC3DTilesLayer(url: .stuttgart3DTiles)
        
        // some comment...
        scene.addOperationalLayer(ogc3DTilesLayer)
```

A very minor suggestion, but I would break this up since it is a simple sample and it is two separate steps in the README. Something along these lines. Your call, though.